tests/hazmat/primitives/test_ec.py

```diff
@@ -182,6 +182,23 @@ def test_encode_point():
     )
 
 
+def test_encode_compressed_point():
+    # secp256r1 point
+    x = int(
+        '233ea3b0027127084cd2cd336a13aeef69c598d8af61369a36454a17c6c22aec',
+        16
+    )
+    y = int(
+        '3ea2c10a84153862be4ec82940f0543f9ba866af9751a6ee79d38460b35f442e',
+        16
+    )
+    pn = ec.EllipticCurvePublicNumbers(x, y, ec.SECP256R1())
+    data = pn.encode_point(compressed=True)
+    assert data == binascii.unhexlify(
+        "02233ea3b0027127084cd2cd336a13aeef69c598d8af61369a36454a17c6c22aec"
```

Comment:
Where did this point come from? Is it possible to get some test vectors for compressed points with 02 and 03 prefixes?

Reply:
i got it from the openssl command line, i can test an 03 as well

Reply:
This project does actually test if OpenSSL works 😄 (we're a major piece of their own testing: https://github.com/pyca/cryptography/tree/09403100de2f6f1cdd0d484dcb8e620f1c335c8f) . But in general known answer tests for things like this are very desirable since we're performing a deterministic transformation.

If we could find some compressed points along with their uncompressed form (maybe in botan or cryptopp or some other lib?) that'd be great.

Reply:
i added a larger number of tests and put it in a loop